src/Core/Banshee.WebBrowser/Banshee.WebSource/WebView.cs

```diff
@@ -26,64 +26,89 @@
 // THE SOFTWARE.
 
 using System;
-
-using Gtk;
-
-using Hyena;
-using Hyena.Downloader;
-
+using System.Runtime.InteropServices;
 using Banshee.Base;
-using Banshee.IO;
 using Banshee.WebBrowser;
+using Gdk;
+using GLib;
+using Hyena.Downloader;
+using EventArgs = System.EventArgs;
+using EventHandler = System.EventHandler;
+using Log = Hyena.Log;
```

Comment:
Ah, let's not sort usings when they have an order to them, original authors went System, GAC libs, Hyena and then the Banshee ones they were interested in. Can we turn sorting using statements off in the auto-formatter?

This one's just a habit but since aliases are a little different from regular using statements I tend to separate them from the others with a blank line, not sure if you agree on that one though?

Reply:
Let's see what settings there are. I'll update the pending R# PR accordingly.